zfsbootmenu/lib/zfsbootmenu-core.sh

```diff
@@ -1281,7 +1281,7 @@ timed_prompt() {
         delay="${OPTARG:-0}"
         if [ "${delay}" -gt 0 ] >/dev/null 2>&1; then
           :
-        elif [ "${delay}" -le 0 ] >/dev/null 2>&1; then
+        elif [ "${delay}" -lt 0 ] >/dev/null 2>&1; then
```

Comment:
Maybe add an `elif [ "${delay}" -eq 0 ]` block to just `return 0` without printing a debug message.

Also, `s/non-positive/negative/` in the comment on line 1266.